### ISSUE TITLE
CE2: Note Editor: Using ** in a list first the first time drops the first letter of the entry

### ISSUE THREAD

**Issue by TheGitMagician** [YoYoGames/GameMaker-Bugs#5301]
### Description

When creating an unordered list using * in the new note editor you can create an indented entry using **. 

However, in the first entry you create like that the first letter of the entry is dropped. 

So:

\* Test
\*\* Test

is shown as

Test
  est

[contactus.zip](https://github.com/YoYoGames/GameMaker-Bugs/files/14858998/contactus.zip)

### Expected Change

Make the first ** entry show all letters

### Steps To Reproduce

See above

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.400 (Betas)

### Which operating system(s) are you seeing the problem on?

Windows 10

### Are you running GameMaker from inside your Steam library?

No

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by yyalansavage** [comment]
Hmm this looks to be working ok for me on IDE 2024.400.0.532 - perhaps I am missing something?
![5301-markdown](https://github.com/YoYoGames/GameMaker-Bugs/assets/8818217/db36279f-9b4a-4f03-93e5-4d96543c541a)

**Comment by TheGitMagician** [comment]
You are right. I cannot reproduce it now. It seems to have been a side effect of when I got the editor unstable by using ö/ä/ü/ß (which is another ticket I have submitted).

Sorry for the inconvenience! This can be closed.

**Comment by alicemoretti** [comment]
Verified that this does not happen on IDE v2024.400.0.536  Runtime v2024.400.0.555. 

Closing the bug. Thank you.
